app/assets/javascripts/pages/scenario-show-page.js.coffee

```diff
@@ -0,0 +1,15 @@
+class @ScenarioShowPage
+  constructor:() ->
+    @changemodaltext()
+
+  changemodaltext: () ->
+    $('#disable_all').click ->
+      $('.modal-body').text 'Would you like to disable all agents?'
+      $('#disabledfield').val 1
```

Comment:
Might be nice to scope all of these to a class or ID on the form or modal.  So something like `$('#scenario-enable-disable-modal .modal-body').text`, otherwise it could affect other modals.